meetings/2023-03-16.md

```diff
@@ -0,0 +1,117 @@
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* SC
+* HZ
+
+
+### Introductions
+* name: text
+
+---
+
+## Topics
+
+
+### WIP Implementation Feedback
+* SC: We'll allocate some time for implementation feedback or interest to implement. Links to products/projects and demos welcome.
+
+
+### Milestones
+URL: https://github.com/solid/notifications/milestone/1
+
+
+### Possible optimization while using StreamingHTTP channel
+URL: https://matrix.to/#/!WdNSKNKuHrHkWppqfn:gitter.im/$b8VNQ3m096NLrWSjvDdRGm5M1nvMy5Eyq_v3xN-wILg
+
+* RG: The reason we are sending capability URLs is to change protocols.
+* RG: Assume that in an alternate universe, we were designing a notifications protocol with only StreamingHTTP.
+* RG: We have broadly identified the following requirements to start notifications:
+  1. The Receiver needs to communicate that she is Authenticated Alice and not Random Bob.
+  2. The Receiver needs to provide a list of routes (topic resources).
+  3. The Receiver needs to communicate the features for the notification channel.
+* RG: We have identified the following requirements to manage notifications:
+  1. The Receiver needs to update routes and features.
+  2. The Receiver needs to unsubscribe.
+* RG: The argument over the unsubscribe model has got me thinking. Right now, the client subscribes with a Subscription Service using `POST`. What if, for StreamingHTTP, instead of sending back a capability URL, the Server starts streaming a response? A `POST` to the Streaming Service (interim name for the combined endpoint in the new model) can communicate all the information, and the response can begin with the notification channel model (whether in the body or header can be argued) followed by the notifications.
+* RG: One of the models proposed in the modify/unsubscribe model is to use `POST` exclusively to modify and unsubscribe. We can do the same with the Streaming Service, with one of the following outcomes: 
+  1. Changes are acknowledged in the new response; Original stream continues.
+  2. Stream is moved over to the new response.
+* RG: We can use PATCH/DELETE as well if the first message/header to the initial POST provides a management URI.
+* RG: I think this will work just as well and allow one round-trip subscription (Assuming we have minimal discovery information in header, that's a separate issue though) resolving #110.
```

Comment:
```suggestion
* RG: I think this will work just as well and allow one round-trip subscription (assuming we have minimal discovery information in header; that's a separate issue though) resolving #110.
```